### ISSUE TITLE
Signup form: have Location/zipcode field post to the "Where are you located?" field in the Google spreadsheet

### ISSUE THREAD

**Issue by audiodude** [CuriosityHacked/curiosityhacked.org#13]
Currently, the google spreadsheet contains a field that is not present in the original signup form: "Where are you located?" It could be imagined that this field should contain either "United States" or the current value of the Zipcode/Location field when the user is not inside the United States.

This is definitely doable, but will require some Javascript hackery.

**Comment by funrollloops** [comment]
The original form has radio buttons "US" and "Outside," doesn't it?

On Sun, Aug 3, 2014, 11:51 AM Travis Briggs notifications@github.com
wrote:

Currently, the google spreadsheet contains a field that is not present in
the original signup form: "Where are you located?" It could be imagined
that this field should contain either "United States" or the current value
of the Zipcode/Location field when the user is not inside the United States.

This is definitely doable, but will require some Javascript hackery.

—
Reply to this email directly or view it on GitHub
https://github.com/CuriosityHacked/curiosityhacked.org/issues/13.

<img src="
https://ci3.googleusercontent.com/proxy/Az7qymNEKP42sph0sCqUF43D8lM5W7r7LnOao4ggjiU75VHSE5M1-uuW7og4W4-N8mlwECrCD2KzbeO9-jjA42fsHTSU0a-mhWWRoIlzBuOkf6yJ80kYuszAgnap4-fCwC3x-JvCHm5mjQMklABdxvI8bw0uUp79RN8ktFjcA94zRaGVXuweDFo5JSJ2qGQgdsvHoN2tBOiRr5bYgVamELvP9-q3mhyPqvTQ4FYTSE-FgnMlTMeS_z-2kl62b8_1AWG0U-x6Rjv2zWOrCRfJKoNUgRU=s0-d-e1-ft#https://github.com/notifications/beacon/75583__eyJzY29wZSI6Ik5ld3NpZXM6QmVhY29uIiwiZXhwaXJlcyI6MTcyMjcxMTA2NiwiZGF0YSI6eyJpZCI6Mzg2OTExMDJ9fQ==--98c4782e14a2fb6ee32c73c13f31aa81913a4161.gif
">

**Comment by audiodude** [comment]
That's right. Were you expecting that radio button to populate that field? That shouldn't be too hard to add.

I was thinking of something more complex where, if they choose "Outside", the zipcode field becomes a "where are you located?" field. But it should be easy to make that field have the value of US/Outside.

**Comment by funrollloops** [comment]
That sounds fine. The goal is to make it easy to geolocate the responses.
US/Outside seems good enough for now.

**Comment by funrollloops** [comment]
Fixed by changing the Google Form and 549180
